src/patternfly/components/Alert/styles.scss

```diff
@@ -1,68 +1,148 @@
+  // Description
+  --pf-c-alert__description--MarginTop: var(--pf-global--spacer--xs);
+  --pf-c-alert__description--MarginBottom: var(--pf-global--spacer--xs);
+
+  // Title
+  --pf-c-alert__title--FontSize: var(--pf-global--FontSize--md);
+
+  // Action
+  --pf-c-alert__action--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-alert__action--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-alert__action--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-alert__action--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Success
+  --pf-c-alert--m-success__icon--color: var(--pf-global--success-color--200);
+  --pf-c-alert--m-success__icon--BackgroundColor: var(--pf-global--success-color--100);
+  --pf-c-alert--m-success__title--color: var(--pf-global--success-color--200);
+
+  // Danger
+  --pf-c-alert--m-danger__icon--color: var(--pf-global--danger-color--300);
+  --pf-c-alert--m-danger__icon--BackgroundColor: var(--pf-global--danger-color--100);
+  --pf-c-alert--m-danger__title--color: var(--pf-global--danger-color--200);
+
+  // Warning
+  --pf-c-alert--m-warning__icon--color: var(--pf-global--warning-color--200);
+  --pf-c-alert--m-warning__icon--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-alert--m-warning__title--color: var(--pf-global--warning-color--200);
+
+  // Info
+  --pf-c-alert--m-info__icon--color: var(--pf-global--info-color--200);
+  --pf-c-alert--m-info__icon--BackgroundColor: var(--pf-global--info-color--100);
+  --pf-c-alert--m-info__title--color: var(--pf-global--info-color--200);
+
+}
 .pf-c-alert {
-  position: relative;
-  padding: $pf-c-alert--PaddingTop $pf-c-alert--PaddingRight $pf-c-alert--PaddingBottom $pf-c-alert--PaddingLeft;
-  border: $pf-c-alert--BorderWidth solid transparent;
+  display: flex;
+  flex-wrap: nowrap;
+  box-shadow: var(--pf-c-alert--box-shadow);
+
+  &:not(.pf-m-top-aligned) {
+    .pf-c-alert__icon {
+      align-items: center;
+    }
+  }
+
+  &__icon, &__action {
+    flex: 1 1 auto;
+  }
+
+  &__icon {
+    display: flex;
+    padding: var(--pf-c-alert__icon--PaddingTop) var(--pf-c-alert__icon--PaddingRight) var(--pf-c-alert__icon--PaddingBottom) var(--pf-c-alert__icon--PaddingLeft);
+    font-size: var(--pf-c-alert__icon--FontSize);
+    line-height: 0;
+  }
+
+  &__body {
+    flex: 1 1 100%;
+    align-self: center;
+    padding: var(--pf-c-alert__body--padding);
+  }
 
-  &__heading {
-    // Specified to prevent conflicts of changing $headings-color
-    color: inherit;
+  &__description {
+    margin: var(--pf-c-alert__description--MarginTop) 0 var(--pf-c-alert__description--MarginBottom);
   }
 
-  &__link {
-    font-weight: $pf-c-alert__link--FontWeight;
+  &__title {
+    font-size: var(--pf-c-alert__title--FontSize);
   }
 
-  &.pf-is-success {
-    color: $pf-c-alert--is-success--Color;
-    background-color: $pf-c-alert--is-success--BackgroundColor;
-    border-color: $pf-c-alert--is-success--BorderColor;
+  &__action {
+    align-self: flex-start;
```

Comment:
This looks unnecessary.

Reply:
ya you're right, old code I forgot to take out :)